pyrefly/lib/alt/class/class_metadata.rs

```diff
@@ -399,7 +426,7 @@ impl<'a, Ans: LookupAnswer> AnswersSolver<'a, Ans> {
         bases_with_metadata
             .iter()
             .find_map(|(base_class_object, metadata)| {
-                if base_class_object.has_toplevel_qname(
```

Comment:
these diffs are fishy, looks like there was a conflict & might need to revert some of those changes